
agp/intel: add new host bridge id for Q57 system

Add new Host Bridge ID found on a Q57 based system from Positivo. I
don't know what abbreviation id for this would be best, may be Q_HB
instead of UNKNOWN currently used would be better.

Signed-off-by: Herton Ronaldo Krzesinski <herton@mandriva.com.br>

[ rebased for 3.8-rc3 /tmb ]
Signed-off-by: Thomas Backlund <tmb@mageia.org>

 drivers/char/agp/intel-agp.c |    1 +
 drivers/char/agp/intel-agp.h |    1 +
 drivers/char/agp/intel-gtt.c |    2 ++
 files changed, 4 insertions(+)

diff -Nurp linux-3.8-rc3/drivers/char/agp/intel-agp.c linux-3.8-rc3.q57/drivers/char/agp/intel-agp.c
--- linux-3.8-rc3/drivers/char/agp/intel-agp.c	2013-01-10 08:17:09.690594184 +0200
+++ linux-3.8-rc3.q57/drivers/char/agp/intel-agp.c	2013-01-10 08:38:56.629888393 +0200
@@ -902,6 +902,7 @@ static struct pci_device_id agp_intel_pc
 	ID(PCI_DEVICE_ID_INTEL_IRONLAKE_D2_HB),
 	ID(PCI_DEVICE_ID_INTEL_IRONLAKE_M_HB),
 	ID(PCI_DEVICE_ID_INTEL_IRONLAKE_MA_HB),
+	ID(PCI_DEVICE_ID_INTEL_IRONLAKE_UNKNOWN),
 	ID(PCI_DEVICE_ID_INTEL_IRONLAKE_MC2_HB),
 	{ }
 };
diff -Nurp linux-3.8-rc3/drivers/char/agp/intel-agp.h linux-3.8-rc3.q57/drivers/char/agp/intel-agp.h
--- linux-3.8-rc3/drivers/char/agp/intel-agp.h	2013-01-10 08:17:09.690594184 +0200
+++ linux-3.8-rc3.q57/drivers/char/agp/intel-agp.h	2013-01-10 08:38:56.649888839 +0200
@@ -187,6 +187,7 @@
 #define PCI_DEVICE_ID_INTEL_IRONLAKE_D_IG	    0x0042
 #define PCI_DEVICE_ID_INTEL_IRONLAKE_M_HB	    0x0044
 #define PCI_DEVICE_ID_INTEL_IRONLAKE_MA_HB	    0x0062
+#define PCI_DEVICE_ID_INTEL_IRONLAKE_UNKNOWN	    0x0069
 #define PCI_DEVICE_ID_INTEL_IRONLAKE_MC2_HB    0x006a
 #define PCI_DEVICE_ID_INTEL_IRONLAKE_M_IG	    0x0046
 
diff -Nurp linux-3.8-rc3/drivers/char/agp/intel-gtt.c linux-3.8-rc3.q57/drivers/char/agp/intel-gtt.c
--- linux-3.8-rc3/drivers/char/agp/intel-gtt.c	2013-01-10 08:17:09.690594184 +0200
+++ linux-3.8-rc3.q57/drivers/char/agp/intel-gtt.c	2013-01-10 08:38:56.649888839 +0200
@@ -1303,6 +1303,8 @@ static const struct intel_gtt_driver_des
 		&g4x_gtt_driver },
 	{ PCI_DEVICE_ID_INTEL_IRONLAKE_D_IG,
 	    "HD Graphics", &ironlake_gtt_driver },
+	{ PCI_DEVICE_ID_INTEL_IRONLAKE_UNKNOWN,
+	    "HD Graphics", &ironlake_gtt_driver },
 	{ PCI_DEVICE_ID_INTEL_IRONLAKE_M_IG,
 	    "HD Graphics", &ironlake_gtt_driver },
 	{ 0, NULL, NULL }
